moma_launch/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(moma_launch)
+
+find_package(catkin REQUIRED)
+catkin_package()
+
+install(DIRECTORY launch config
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+)
```

Comment:
As discussed, I would remove this package as, for the moment being, is just more confusing then helpful